keel-sql/src/main/resources/db/changelog/20190924-rename-server-group-back-to-cluster-sigh.yml

```diff
@@ -0,0 +1,46 @@
+databaseChangeLog:
+  - changeSet:
+      id: rename-server-group-back-to-cluster-sigh
```

Comment:
I think this is not going to work given the structure of a `ClusterSpec` is different to old `ServerGroupSpec`. I think we're actually going to have to delete any server group  resources and recreate them as clusters.